### PR TITLE
Issue 440 siblings of badly timed elements

### PR DESCRIPTION
**Pull request by nigelmegitt**
Pull request for fixing issue #440.

### CODE REVIEW

File: ebu_tt_live/bindings/validation/timing.py [hunk 1]
```diff
@@ -43,6 +43,13 @@ def _pre_init_variables(self, dataset, element_content):
         self._computed_end_time = None
         self._semantic_dataset = dataset
 
+    def _exclude_badly_timed_elements_filter(self, value, element):
```
Comment (tomrosier) [inline ebu_tt_live/bindings/validation/timing.py]: Could this function be short handed?
`return True if not (element.begin is not None and element.end is not None and element.begin >= element.end) False` or something like that

File: ebu_tt_live/bindings/validation/timing.py [hunk 2]
```diff
@@ -176,7 +183,10 @@ def _semantic_postprocess_timing(self, dataset, element_content):
             # This requires calculation based on the timings in its children.
 
             # All timing containers are complexTypes so we can call orderedContent safely
-            children = filter(lambda item: isinstance(item, TimingValidationMixin), [x.value for x in self.orderedContent()])
+            # but we don't want to bother with explicitly badly timed elements so filter
+            # them out.
+            children = filter(lambda item: isinstance(item, TimingValidationMixin) \
```
Comment (tomrosier) [inline ebu_tt_live/bindings/validation/timing.py]: Not a fan with how long this line is.

File: testing/bdd/features/timing/computed_times.feature [hunk 3]
```diff
@@ -11,6 +11,7 @@ Feature: Computed times computation
   # SPEC-CONFORMANCE: R16 R17 R132
   Scenario: Computed times of a document
     Given an xml file <xml_file>
+    And l is <l>
```
Comment (tomrosier) [inline testing/bdd/features/timing/computed_times.feature]: can this be more descriptive?

File: ebu_tt_live/bindings/validation/timing.py (hunk 1 above)
Comment (nigelmegitt) [inline ebu_tt_live/bindings/validation/timing.py]: I don't think Python 2.7 allows that. But I can change it to:

```python
return (element.begin is None or element.end is None or element.end > element.begin)
```

File: ebu_tt_live/bindings/validation/timing.py (hunk 2 above)
Comment (nigelmegitt) [inline ebu_tt_live/bindings/validation/timing.py]: Me neither. Looks like the only easy thing to do is to split the second argument to `filter(function, iterator)` onto a second line.

File: testing/bdd/features/timing/computed_times.feature (hunk 3 above)
Comment (nigelmegitt) [inline testing/bdd/features/timing/computed_times.feature]: Yes, making it so...

File: ebu_tt_live/bindings/validation/timing.py (hunk 1 above)
Comment (tomrosier) [inline ebu_tt_live/bindings/validation/timing.py]: I think that would be the right thing todo :)

File: ebu_tt_live/bindings/validation/timing.py (hunk 2 above)
Comment (nigelmegitt) [reply ebu_tt_live/bindings/validation/timing.py]: OK, this was done though it is not showing as "outdated" on the GitHub pull request page strangely - @tomrosier if you're happy with it please could you "approve" in a review?

**Pull request merged**